### DISCUSSION TITLE
new version does not download with the name of the file,

### DISCUSSION THREAD

**Discussion by sam-github-sam** [aclap-dev/video-downloadhelper#194] [Help and Q&A]
### Your question

new version does not download with the name of the file, uses 1111111.mp4 or aaaaaaa.vid from sites where it used to download with the name shown in the file list. What is going on ?

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/117.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\mswas",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by Kobra299** [comment]
I asked the same thing and got an answer https://github.com/aclap-dev/video-downloadhelper/discussions/176 that it will be fixed in update coming soon

**Comment by sam-github-sam** [comment]
Cheers. Do you know if they addressed the fact that some files show as .mp4 in the file list but then download as .vid ?

**Reply by paulrouget** [reply]
Yes, this is addressed. 8.2.1 should fix that.
